src/Symfony/Component/String/AbstractString.php

```diff
@@ -574,14 +574,14 @@ public function toBinary(string $toEncoding = null): BinaryString
         return $b;
     }
 
-    public function toGrapheme(): GraphemeString
+    public function toGrapheme(): UnicodeString
     {
-        return new GraphemeString($this->string);
+        return new UnicodeString($this->string);
     }
 
-    public function toUtf8(): Utf8String
+    public function toUtf8(): CodePointString
```

Comment:
should it be renamed to `toCodePoint` ?

Reply:
updated to `toCodePointString` for clarity